projects/step-frontend/src/lib/modules/admin/admin.module.ts

```diff
@@ -113,7 +113,7 @@ export class AdminModule {
 
     this._viewRegistry.registerRoute(
       {
-        path: 'myaccount',
+        path: 'my-account',
```

Comment:
@lukasz-lepa 
There are still places in project, that references to `myaccount` link

Reply:
Fixed.